generate_diagrams.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+directory="./docs/model_diagrams/"
+
+if [ ! -d "$directory" ]; then
+    # If the directory doesn't exist, create it
+    mkdir -p $directory
+fi
+
+
+SETTINGS_MODULE="$1"
+
+shift
+
+declare -A unique_apps
+
+MIGRATIONS=$@
```

Comment:
Won't this be empty?

Reply:
No it shouldn't. The hook passes the migration file names as args (at least testing locally on the hub it does. It should behave the same way here)

Reply:
I see, then presumably it is passing all changed files and you need to filter out those that are not from migrations? Or does the pre-commit `files` parameter do that filtering for you?

Reply:
The pre-commit `files` param does it